crates/sui/src/keytool.rs

```diff
@@ -43,8 +43,14 @@ pub enum KeyToolCommand {
     },
     /// This reads the content at the provided file path. The accepted format can be
     /// [enum SuiKeyPair] (Base64 encoded of 33-byte `flag || privkey`) or `type AuthorityKeyPair`
-    /// (Base64 encoded `privkey`). It prints its Base64 encoded public key and the key scheme flag.
-    Show { file: PathBuf },
+    /// (Base64 encoded `privkey`).
+    /// If base_58_no_key_scheme_flag is true, print its Base58 encoded publuc key. Otherwise,
```

Comment:
publ**i**c

Reply:
> I feel a bit hesitant (security-wise) to approve base58 of private key material for two reasons: a) our base58 implementation might not be constant_time (while our base64 library was specifically chosen to be). This is one of the main reasons the industry prefers base64 for key material (ie the whole TLS space with certs etc). b) base58 is expensive in large inputs, if this will work for multi-sig too, then it might take a while.
> 
> What is the main reason we need to also print in base58?

tagged you offline